system-test/spanner.ts

```diff
@@ -3889,7 +3894,7 @@ describe('Spanner', () => {
           let err;
 
           try {
-            await txn.batchUpdate(null);
+            await txn.batchUpdate([]);
```

Comment:
Why are we changing this test? If it is a type issue then you should cast it to `unknown` or `{}`.